v1/diff_test.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package json_test
+package json
```

Comment:
I might prefer to use external test packages in general or by default, but I don't hold that strongly.

Reply:
Unfortunately, I lose access to the private `knownTestFailure` function. We can fix this after the work is done.